evap/staff/templates/staff_semester_grade_reminder.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'staff_semester_base.html' %}
+
+{% load evaluation_filters %}
+
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="breadcrumb-item">{% trans 'Grade Publish Reminder' %}</li>
+{% endblock %}
+
+{% block content %}
+    {{ block.super }}
+
+    <h3>
+        {{semester.name}}
+    </h3>
+
+    <div class="card">
+        <div class="card-body">
+            {% for responsible, courses, delegates in responsible_list %}
```

Comment:
`delegates` is unused. @janno42 is there any need to display that? if not, @LeaGe can you remove that here and in the view?